packages/Toaster/src/Toast/ToastContainer/ToastContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { IToastContainerProps } from "../types/container/container";
+
+const placements = {
+  "bottom-center": { bottom: 0, left: "50%", transform: "translateX(-50%)" },
+  "bottom-left": { bottom: 0, left: 0 },
+  "bottom-right": { bottom: 0, right: 0 },
+  "top-center": { top: 0, left: "50%", transform: "translateX(-50%)" },
+  "top-left": { top: 0, left: 0 },
+  "top-right": { top: 0, right: 0 }
+};
```

Comment:
Dont know if theres a const file but this should be there

Reply:
agree 👍